internal/modelcontroller/model_controller.go

```diff
@@ -220,7 +222,7 @@ func (r *ModelReconciler) vLLMPodForModel(m *kubeaiv1.Model, profile ModelConfig
 		{
 			// TODO: Conditionally set this token based on whether
 			// huggingface is the model source.
-			Name: "HUGGING_FACE_HUB_TOKEN",
```

Comment:
Why change in vLLM and FasterWhisper?

Reply:
Because we should be using HF_TOKEN everywhere. 

Using HUGGING_FACE_HUB_TOKEN is deprecated and may be removed in future releases.

Reply:
I can do this in a separate PR if you prefer.

Reply:
No need to make another PR, just wondering